webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -1431,6 +1435,10 @@ var CURRENT_GROUP_DATA={};
                 if (optns.metadata) {
                     userPxy.metadata = optns.metadata;
                 }
+                if (window.applozic.PRODUCT_ID == APPLOZIC) {
+                    document.getElementById("mck-tab-option-panel").style.display = null;
```

Comment:
@kumarabhinav002 any particular reason to set the `style.display` to `null`?

Reply:
To clear mck-tab-option-panel display attribute. In case of kommunicate it is set to display:none !important.

Reply:
@kumarabhinav002 then please change it to one of the valid properties of `display` and not `null`.